From june4 meeting suggestions

GeorgeKerscher opened the pull request:
Following the TIES call on June 4, I made the suggested changes. I tried to make separate commits for the various parts. I hope it came out like that!

1. Added a link to our issue tracker for communications.
2. Added more clarification about verbosity. I decided to continue to use that word and associate it with all, most, some, and none.
3. In each entry I reference all, most, some, or none, and I added all that applied, i.e. high is some, most, and all, and medium is some and all.
4. I moved the complementary to the end and noted why it is there.
5. I changed the spelling of acknowledgments to mach the doc-acknowledgment. Note that in the DPUB aria specification that we mix US English and British English speling. Prolog is US and prologue is British, and epilog is us and epilogue is British.

clapierre commented on dpub-aria-testing/content/DPUB-ARIA-tests.xhtml:
```diff
@@ -6,46 +6,47 @@
-			This text in a <code>div</code> element has the  role of complementary.
-		</div>
+
+<p>Note: a verbosity selection of"some" includes critical and high importance. A verbosity selection of "most includes critical, high, and medium.</p> 
```
space between of and "some" needed

clapierre reviewed (commented):
George, I thought we also discussed not using "verbosity" alone and instead use something more specific  like Document Semantics Verbosity or something like that?

GeorgeKerscher commented:
Yes, we discussed this, but it seems that when adding all, most, some, and none that verbosity worked well. Should I add document semantics verbosity everywhere?

clapierre commented:
Good question,, what do others think?

GeorgeKerscher commented:
In the latest commit, I established that document semantics verbosity as a term and use it in association with the four values.

I fixed two problems with a missing space character.

I also removed the single letter navigation suggestion of p, because in the latest beta release of NVDA the single letter navigation of p is associated with next paragraph. I left the single letter navigation suggestion, but did not suggest a character.
